apollo-router/src/services/snapshots/apollo_router__services__supergraph_service__tests__errors_from_primary_on_deferred_responses.snap.new

```diff
@@ -0,0 +1,17 @@
+---
+source: apollo-router/src/services/supergraph_service.rs
+assertion_line: 766
+expression: stream.next_response().await.unwrap()
+---
+{
+  "data": null,
+  "errors": [
+    {
+      "message": "couldn't find mock for query {\"query\":\"{computer(id:\\\"Computer1\\\"){id errorField}}\"}",
+      "extensions": {
+        "code": "FETCH_ERROR"
+      }
```

Comment:
I don't think this is what is expected.
This often means the query planner was updated and the mock was used to work with `{errorField id}`

⚠️ i think this needs to be addressed before merging

Reply:
agree, not sure how this made it in - thought i fixed that up.

Reply:
If possible can we make the test panic if it detects a missing mock?

Reply:
what do you mean?

Reply:
If the snapshot contains `couldn't find mock for query` it should hard fail the test.